cli/cmd/inject.go

```diff
@@ -318,7 +323,9 @@ func (resourceTransformerInject) generateReport(reports []inject.Report, output
 	}
 
 	for _, r := range reports {
-		if b, _ := r.Injectable(); b {
+		if r.Kind == k8s.Service {
+			output.Write([]byte(fmt.Sprintf("service \"%s\" skipped\n", r.Name)))
```

Comment:
As mentioned in the description, this still results in `service "..." skipped` since we are skipping injection in the `inject` command path:

```
$ linkerd inject test/integration/testdata/smoke_test.yaml
...
deployment "smoke-test-terminus" injected
service "smoke-test-terminus-svc" skipped
deployment "smoke-test-gateway" injected
service "smoke-test-gateway-svc" skipped
```